doc/developer/frr-release-procedure.rst

```diff
@@ -80,30 +80,39 @@ FRR Release Procedure
          .
            * Your Changes Here
 
-6. Change main version number:
+6. Commit the changes, adding the changelog to the commit message. Follow all
+   existing commit guidelines. The commit message should be akin to::
 
-    - Edit :file:`configure.ac` and change version in the ``AC_INIT`` command
-      to ``<version>``
+      debian, redhat: updating changelog for new release
 
-7. Commit the changes, adding the changelog to the commit message. Follow all
-   existing commit guidelines.
+7. Create a new branch based on ``master``, cherry-pick the commit made in step
```

Comment:
I'd delay this step until we do testing further down to make sure everything is working in the release branch, just in case there are any syntax/formatting errors  in changelog files.